subfunctions/f_CaBBI_QGIF

```diff
@@ -0,0 +1,56 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% QGIF evolution function:
+%
+%                   [fx] = f_CaBBI_QGIF(Xt,Theta,Isyn,inF)
+%
+% this function contains the equations and parameters of the QGIF neuron
+% model, and the over-time integration scheme
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  
+
+function [fx] = f_CaBBI_QGIF(Xt,Theta,Isyn,inF)
```

Comment:
I think it might be better that we currently skip computation of derivatives analytically, for this and other functions. This is (mainly) because, as you mentioned, the performance is not an issue currently.